Update Pangram.java using Java Collections

Ankit-Saha08 opened the pull request:
A simple separate function isPangramOrNot(String s) has been created which implements the Pangram checking of a string using Java Collection Framework approach.

<!-- For completed items, change [ ] to [x] -->

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

Ankit-Saha08 commented:
I request the maintainer(s) to merge my request using "hacktoberfest-accepted" label.

debasishbsws commented on src/main/java/com/thealgorithms/strings/Pangram.java:
```diff
@@ -15,6 +17,22 @@ public static void main(String[] args) {
+     * Checks if a String is considered a Pangram
+     *
+     * @param s The String to check
+     * @return {@code true} if s is a Pangram, otherwise {@code false}
+     */
+    public static boolean isPangramOrNot(String s) {
+        // an approach using Java Collection Framework
```
```suggestion
     * @return {@code true} if s is a Pangram, otherwise {@code false}
     */
     // alternative approach using Java Collection Framework
    public static boolean isPangramOrNot(String s) {
```

Ankit-Saha08 replied:
I have committed the above stated changes

debasishbsws commented:
Looks like you haven't add test for your function. (It is always better to add test you know)
https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/strings/PangramTest.java
Add some test for you method too. you can use the same example just with your method call.
and `isPangramUsingSet(String s)` is a better name

Ankit-Saha08 commented:
Ok, I will update the method name isPangramUsingSet(String s) and also add some tests for my method.

Ankit-Saha08 commented:
Hi, I have updated the function name to isPangramUsingSet(String s) and also added tests for my function.

P.S. - I would be grateful if my contribution is accepted and merged using "hacktoberfest-accepted" label.

debasishbsws commented:
Add the test in this barnch.

Ankit-Saha08 commented:
Hey, I am facing an issue while trying to commit the changes in PangramTest.java file. I am unable to find this PR branch while trying to commit the changes in the test file. What should I do now?

Like this current branch name is - "patch-4" but while trying to select the branch before committing it isn't showing up there. Looking forward for some gainful insight on resolving this

Ankit-Saha08 commented:
Hii, I have updated the test cases in PangramTest.java file in this current PR branch itself. 

**P.S. - Looking forward to having my contribution merged and accepted.**

debasishbsws reviewed (approved):
Looks Good. Thanks
